src/lambda/vue/samInvokeFrontend.ts

```diff
@@ -168,18 +172,17 @@ export default defineComponent({
             handler(newval: AwsSamDebuggerConfigurationLoose) {
                 vscode.setState(
                     Object.assign(vscode.getState() ?? {}, {
-                        launchConfig: newval,
+                        launchConfig: JSON.parse(JSON.stringify(newval)),
```

Comment:
These clones aren't strictly needed at the moment but it's better to safe-guard now since these types of bugs are difficult to track now. Basically, anything related to Vue <--> VS Code should use this pattern to rip out the reactivity in case VS Code behavior w.r.t proxies changes.

Reply:
a good thing to mention in the commit message :)

Reply:
Added a comment but I'll add some extra info in the squash merge commit message too